p2p/message_test.go

```diff
@@ -222,6 +248,40 @@ func TestEncodeBlockRequestMessage_BlockNumber(t *testing.T) {
 	}
 }
 
+func TestDecodeBlockRequestMessage_BlockNumber(t *testing.T) {
+	encMsg, err := common.HexToBytes("0x0107000000000000000101010000000000000001fd19d9ebac759c993fd2e05a1cff9e757d8741c2704c8682c15b5503496b6aa1010101000000")
```

Comment:
Please include values used to create these

Reply:
done